src/reprepro_updater/helpers.py

```diff
@@ -77,18 +77,59 @@ def _run_update_command(repo_dir, distro, commit):
     return try_run_command(update_command)
 
 
+def _get_dependent_packages(repo_dir, distro, arch, package):
+    """ Return a list of packages which were detected as dependant by reprepro.
+    This only returns direct dependencies. """
+
+    reprepro_command = [
+        'reprepro', '-V', '-b', repo_dir,
+        '-T', 'deb',
+        'listfilter', distro,
+        "Package (% ros-* ), " +
+        "Architecture (== " + arch + " ), " +
+        "( Depends (% *" + package + "[, ]* ) " +
+        "| Depends (% *" + package + " ) )"]
+
+    try:
+        output = subprocess.check_output(reprepro_command)
+    except subprocess.CalledProcessError as ex:
+        print "Execution of [%s] Failed:" % reprepro_command, ex
```

Comment:
I didn't mean that. I think new code should be written in a Python 3 compatible way. There is simply no need to write code which is Python 2 only.